src/components/ActionBar/ActionBar.tsx

```diff
@@ -33,7 +33,7 @@ export function ActionBar(props: ActionBarProps) {
         </Button>
         <ImageSelectButton
           hidden={!!props.assets.length}
-          disabled={props.selectedAsset?.src === ''}
+          disabled={!props.selectedAsset}
```

Comment:
Is there no chance that selected asset is set an as empty string? Double checking we don't have a regression here by relying it be defined or undefined.

Reply:
The selected asset should always be either an object containing the asset's properties, or explicitly set as undefined. Additionally, as far as I can tell, the `src` can also never be an empty string unless manually overridden, as even with a empty asset name and empty destination path (e.g. one of our testing assets), it still includes the source path.